typespec/radius/v1/resources.tsp

```diff
@@ -62,6 +62,23 @@ model ApplicationScopedResource {
   status?: ResourceStatus;
 }
 
+@doc("Base properties of a Global-scoped resource")
+model GlobalScopedResource {
+  @doc("Fully qualified resource ID for the environment that the application is linked to")
+  environment?: string;
+
+  @doc("Fully qualified resource ID for the application")
+  application?: string;
```

Comment:
Can a globally scoped resource still have these set?

Reply:
My question is should they be optional or not exist at all?

Reply:
Here we are just making application as optional so user can skip providing env/app to make it global scope. 
just how we have application as optional in environment scoped resources.